Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM messense/rust-musl-cross:x86_64-musl AS build
+
+LABEL MAINTAINER="engineering@spicelabs.io"
```

Comment:
Have you set up this alias?

Reply:
fixing for ext-engineering instead as discussed.